ui/monk/windwalker/sim.ts

```diff
@@ -163,6 +184,16 @@ export class WindwalkerMonkSimUI extends IndividualSimUI<Spec.SpecWindwalkerMonk
 				getEPDefaults: (player: Player<Spec.SpecWindwalkerMonk>) => {
 					return getActiveEPWeight(player, this.sim);
 				},
+				updateSoftCaps: (softCaps: StatCap[]) => {
+					if (hasTwoHandMainHand(player)) {
+						const hasteSoftCap = softCaps.find(v => v.unitStat.equalsPseudoStat(PseudoStat.PseudoStatMeleeHastePercent));
+						if (hasteSoftCap) {
+							// Two-Handed Windwalkers need to adjust for Way of the Monk 40% Melee Haste
+							hasteSoftCap.breakpoints = hasteSoftCap.breakpoints.map(v => v + 40);
```

Comment:
Is this really an additive buff? All other non-Rating Haste effects are multiplicative.